draft-js-hashtag-plugin/CHANGELOG.md

```diff
@@ -5,6 +5,12 @@ This project adheres to [Semantic Versioning](http://semver.org/).
 
 ## To Be Released
 
+## 1.0.1
```

Comment:
If we follow http://semver.org/ I would consider this a MINOR change and suggest 1.1.0. What are your thoughts?